### PR TITLE
BUGFIX: Numerous drag-and-drop issues

### PR DESCRIPTION
**Pull request by unclecheese**
Fixes: https://github.com/silverstripe/silverstripe-graphql/issues/323

### CODE REVIEW

**Comment by Cheddam** [comment]
@unclecheese How confident are you that these are the only remaining instances of incorrect casing? 😅

**Comment by unclecheese** [comment]
Yeah, I think about this all the time!

It's quite a tricky thing to root out, because find and replace is pretty easily defeated by things like destructuring. But on the other hand, the unit tests and Behat tests are passing, and we have a team using it on a real project, so I suspect any remaining instances won't have many places to hide anymore. :-)

File: client/src/components/ElementEditor/ElementEditor.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ class ElementEditor extends PureComponent {
    * @param afterId
    */
   handleDragEnd(sourceId, afterId) {
+    console.log('drag end', sourceId, afterId);
```
Comment (ScopeyNZ) [inline client/src/components/ElementEditor/ElementEditor.js:58]:
```suggestion
```

**Review by ScopeyNZ — COMMENTED** [review]
I'm glad it's related to the GraphQL 4 update and not the code I wrote 😅

**Comment by unclecheese** [comment]
There should be a special Github rating for how many console.logs you've discovered.

**Comment by chillu** [comment]
@unclecheese Looks like `bundle.js` is outdated: https://travis-ci.org/github/dnadesign/silverstripe-elemental/jobs/744730093#L1018

**Comment by michalkleiner** [comment]
clippy Looks like `bundle.js` is outdated

**Pull request merged**